CONTRIBUTIONS.md

```diff
@@ -97,6 +97,9 @@ A lot of people are excited about building a JSON API in Servant and everyone se
   - Investigate the tradeoff of doing all the implementation in haskell vs. interface, via haskell, to html templates.
   - Investigate how to integrate with existing jQuery widgets (calendar, accordion, search/sort tables, editable grids, etc)
```

Comment:
I feel that the question of implementing all the markup directly in reflex vs using templates has been solved: I absolutely suggest using a quasiquoter, as we did.

The jQuery widgets probem is indeed more complex: probably it requires javascript ffi, that I'm learning only now. So I definitely don't have an answer for jQuery widgets integration.